mlir/include/mlir/Dialect/EmitC/Transforms/TypeConversions.h

```diff
@@ -0,0 +1,17 @@
+//===- TypeConversions.h - Convert signless types into C/C++ types -------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_DIALECT_EMITC_TRANSFORMS_TYPECONVERSIONS_H
+#define MLIR_DIALECT_EMITC_TRANSFORMS_TYPECONVERSIONS_H
+
+namespace mlir {
+class TypeConverter;
+void populateEmitCSizeTypeConversions(TypeConverter &converter);
```

Comment:
I assume this should be
```suggestion
void populateEmitCSizeTTypeConversions(TypeConverter &converter);
```